test/extended/util/test.go

```diff
@@ -415,7 +415,7 @@ var (
 
 			// Test fails on platforms that use LoadBalancerService and HostNetwork endpoint publishing strategy
 			`\[Conformance\]\[Area:Networking\]\[Feature:Router\] The HAProxy router should set Forwarded headers appropriately`, // https://bugzilla.redhat.com/show_bug.cgi?id=1752646
-
+			`\[Conformance\]\[templates\] templateinstance impersonation tests should pass impersonation creation tests`,         // https://bugzilla.redhat.com/show_bug.cgi?id=1752910
```

Comment:
what happened to bug 1731222?   it looks like it was fixed, so shouldn't we be re-enabling this test?

Reply:
also not clear why this is a preferred way to skip the test over how it was being done previously?

Reply:
(can ignore my second comment, I see your PR summary, though that is weird).

Reply:
I know very little of the status of the template tests, but what I do know is that this test was already disabled on all platforms, which is what this was for:
```go
g.Skip("Bug 1731222: skip template tests until we determine what is broken")
```

do you know anyone who may be familiar with this test suite who could comment?

Reply:
it's owned by @openshift/sig-developer-experience.  Given that the bug being referenced as the reason for disabling the test, has been closed, they should investigate re-enabling the test or why it is still failing (And open a new bug).

(but if you need to do this PR in the meantime to make progress, that's fine, i just don't want the test to be forever disabled as we lose track of the history of why it was disabled/plans to re-enable it)

Reply:
OpenStack needs its tests green to GA with 4.2, so if we dont hear from them in a few hours lets disable it.